server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/cucumber/CassandraUploadCucumberTest.java

```diff
@@ -30,4 +32,15 @@
                 tags = {"~@Ignore"},
                 strict = true)
 public class CassandraUploadCucumberTest {
+
+    @BeforeClass
+    public static void init() {
+        CucumberCassandraSingleton.cassandraServer.start();
+    }
+
+    @AfterClass
+    public static void after() {
+        CucumberCassandraSingleton.cassandraServer.stop();
```

Comment:
you don't clean tables between each test?

Reply:
that's the whole point : delete stuff slower than creating a new database.

Reply:
I mean: if you don't restart cassandra between each test, how do you ensure the data is clean?

Reply:
every database has a different name